services/hack/uploadSubmission.js

```diff
@@ -0,0 +1,45 @@
+const AWS = require('aws-sdk');
+AWS.config.update({
+    region: "ap-southeast-2",
+    accessKeyId: process.env.AWS_ACC_KEY_ID,
+    secretAccessKey: process.env.AWS_SECRET
+  });
+
+export const upload = async (event, context) => {
+  const s3 = new AWS.S3({ signatureVersion: "v4" });
+  const { fileName } = JSON.parse(event.body);
+
+  console.log(fileName);
+
+  var fileToUpload = {
+    Bucket: 'hack-zero-submission-files',
+    Key: fileName,
+    ContentType: "multipart/form-data",
+    Expires: 30000,
+  };
+
+  try {
+    // Creating the presigned Url
+    const preSignedURL = await s3.getSignedUrl("putObject", fileToUpload);
+    let returnObject = {
+        statusCode: 200,
+        headers: {
+            "access-control-allow-origin": "*"
```

Comment:
We need to raise a task to fix these wildcard cors headers.

Reply:
wait, this is environment variabled already... `process.env.ACCESS_CONTROL_ALLOW_ORIGIN`

Reply:
Ah yep, just forgot switch it out